pkg/cache/cache.go

```diff
@@ -68,29 +68,48 @@ type ResourceCacheArgs struct {
 
 	// ObjectType is the type of object which is to be stored in this cache.
 	ObjectType reflect.Type
+
+	ReconcilerConfig ReconcilerConfig
+}
+
+// ReconcilerConfig contains configuration for the periodic reconciler.
+type ReconcilerConfig struct {
+	// DisableUpdateOnChange disables the queuing of updates when the reconciler
+	// detects that a value has changed in the datastore.
+	DisableUpdateOnChange bool
```

Comment:
What about dropping the Disable on each of these variables?  With Disable then if the variable is false you basically have a double negative which I know I have been confused by before.  Of course that means the default will change.

Reply:
I think it's nice to have the default behavior be the "normal" or expected behavior.  Removing the "Disable" bit would mean more code elsewhere to set these all to "false" by default, so I'd vote for leaving like this.